weatherman/utils.py

```diff
@@ -0,0 +1,143 @@
+import calendar
+from typing import Tuple
+import weather
+
+
+get_month_name = lambda month: calendar.month_name[month][:3]
+
+
+def get_annual_report_of_temperature_and_humidity(year: int) -> Tuple:
+    """Returns the annual report (temperature and humidity)
+
+    Args:
+        year (int):year of which you want annual report
+
+    Returns:
+        Tuple: containing every month maximum temperature,
+               minimum temperature, maximum humidity and 
+               minimum humidity 
+    """
+    
+    yearly_data = weather.weatherman[year]
+    every_month_max_temp = []
+    every_month_min_temp = []
+    every_month_max_humidity = []
+    every_month_min_humidity = []
+    
+    for month in range(1, 13):
+        try:
+            monthly_data = yearly_data[get_month_name(month)] 
+            day_important_information = {}
```

Comment:
Can we rename this to `daily_weather_data`?

Reply:
`day_important_information = {
"everyday_max_temp" = [],
"everyday_min_temp" = [],
}`